external/fv3fit/tests/emulation/test_keras.py

```diff
@@ -112,3 +114,26 @@ def test_model_score_no_outputs():
 
     with pytest.raises(ValueError):
         score_model(model, data)
+
+
+def test_jacobians():
+    sample = {
+        "a": tf.random.normal((100, 5)),
+        "b": tf.random.normal((100, 5)),
+    }
+    sample["field"] = sample["a"] + sample["b"]
+
+    profiles = {
+        name: tf.reduce_mean(sample[name], axis=0, keepdims=True) for name in ["a", "b"]
+    }
+
+    inputs = {"a": tf.keras.Input(5), "b": tf.keras.Input(5)}
+    out = tf.keras.layers.Lambda(lambda x: {"field": x["a"] + x["b"]})(inputs)
+    model = tf.keras.Model(inputs=inputs, outputs=out)
+
+    jacobians = standardize_jacobians(get_jacobians(model, profiles), sample,)
+
+    assert set(jacobians) == {"field"}
+    assert set(jacobians["field"]) == {"a", "b"}
+    for j in jacobians["field"].values():
+        assert j.shape == (5, 5)
```

Comment:
Required. You should check the actual values:
```suggestion
        n = 5  # maybe move to the top of the test
        np.testing.assert_array_almost_equal(jacobians["field"]["a"], np.eye(n))            
        np.testing.assert_array_almost_equal(jacobians["field"]["b"], np.eye(n))

```